examples/effect_test/Filter_LowPass/sketch.js

```diff
@@ -0,0 +1,78 @@
+/**
+ *  Example: Apply a p5.LowPass filter to a p5.SoundFile.
+ *  Visualize the sound with FFT.
+ *  Map mouseX to the the filter's cutoff frequency
+ *  and mouseY to resonance/width of the a BandPass filter
+ */
+
+var soundFile;
+var fft;
+
+var description = 'loading';
+var p;
+
+var filter, filterFreq, filterRes;
+
+function preload() {
+  soundFormats('mp3', 'ogg');
+  soundFile = loadSound('../../files/beat');
+  console.log("hi");
+}
+
+function setup() {
+  createCanvas(710, 256);
+  fill(255, 40, 255);
+
+  // loop the sound file
+  soundFile.loop();
+
+  //filter = new p5.LowPass();
+  
+  //filter = new p5.Filter();
+  filter = new p5.LowPass();
+  
+
+
+  // Disconnect soundfile from master output.
+  // Then, connect it to the filter, so that we only hear the filtered sound
+  soundFile.disconnect();
+  soundFile.connect(filter);
+
+  fft = new p5.FFT();
+
+  // update description text
+  p = createP(description);
+  var p2 = createP('Draw the array returned by FFT.analyze( ). This represents the frequency spectrum, from lowest to highest frequencies.');
+}
+
+function draw() {
+  background(30);
+
+
+  // Map mouseX to a the cutoff frequency for our lowpass filter
+  filterFreq = map (mouseX, 0, width, 10, 22050);
```

Comment:
we could add constraint(0, 22050) to get rid of a warning message in the console when we try to set frequency above 22050